plugins/task-plugin/src/variable/project-path-variable-resolver.ts

```diff
@@ -54,7 +54,7 @@ export class ProjectPathVariableResolver {
             }
         } else {
             // get project folder from workspace folders, first folder in workspaceFolders is .theia
-            const folders = fs.readdirSync(this.projectsRoot, {withFileTypes: false});
+            const folders = fs.readdirSync(this.projectsRoot, { withFileTypes: false });
```

Comment:
maybe we can omit this change for this PR ?

Reply:
I recall I have added it since some pre-build checks were failing